src/components/application_manager/rpc_plugins/sdl_rpc_plugin/include/sdl_rpc_plugin/commands/mobile/subscribe_button_request.h

```diff
@@ -34,8 +34,8 @@
 #ifndef SRC_COMPONENTS_APPLICATION_MANAGER_RPC_PLUGINS_SDL_RPC_PLUGIN_INCLUDE_SDL_RPC_PLUGIN_COMMANDS_MOBILE_SUBSCRIBE_BUTTON_REQUEST_H_
 #define SRC_COMPONENTS_APPLICATION_MANAGER_RPC_PLUGINS_SDL_RPC_PLUGIN_INCLUDE_SDL_RPC_PLUGIN_COMMANDS_MOBILE_SUBSCRIBE_BUTTON_REQUEST_H_
 
-#include "application_manager/commands/command_request_impl.h"
 #include "application_manager/application_impl.h"
+#include "application_manager/commands/command_request_impl.h"
```

Comment:
@mked-luxoft Wrong includes order. It was right before

Reply:
@AByzhynar fixed in2b001c6014447a350d41a2bcd1e834fd2c83057a